common.go

```diff
@@ -250,6 +250,20 @@ func computePayloadHash(version Version, headerHash headerHash, nonce Nonce, cip
 	return sliceToByte64(h)
 }
 
+func computeSigncryptionSignatureInput(headerHash headerHash, nonce Nonce, isFinal bool, chunkPlaintext []byte) []byte {
+	signatureInput := []byte(signatureEncryptedString)
+	signatureInput = append(signatureInput, headerHash[:]...)
+	signatureInput = append(signatureInput, nonce[:]...)
```

Comment:
I could go either way. If we take them out, we'll want to have some loud comments in `nonceForChunkSigncryption` that we're relying on the exact inputs there for the signature part, and that we need to be super careful if we ever change them.